src/chemistry0d.cpp

```diff
@@ -361,6 +361,9 @@ void CanteraGas::setOptions(const ConfigOptions& options)
 
 void CanteraGas::initialize()
 {
+    // suppress thermo warnings to simplify output. Without this warnings printed for every thread created
+    Cantera::suppress_thermo_warnings();
+
```

Comment:
I think users deserve the thermo warnings they get.

Reply:
I agree that the thermo warnings should be generated at least once, but there's no need to generate them every time a new thread is created. After this change, the warnings are output a single time from the top level python.